extensions/Worker.Extensions.CosmosDB/CosmosDBTriggerAttribute.cs

```diff
@@ -9,88 +9,79 @@ namespace Microsoft.Azure.Functions.Worker
     public sealed class CosmosDBTriggerAttribute : TriggerBindingAttribute
     {
         /// <summary>
-        /// Triggers an event when changes occur on a monitored collection
+        /// Triggers an event when changes occur on a monitored container
         /// </summary>
-        /// <param name="databaseName">Name of the database of the collection to monitor for changes</param>
-        /// <param name="collectionName">Name of the collection to monitor for changes</param>
-        public CosmosDBTriggerAttribute(string databaseName, string collectionName)
+        /// <param name="databaseName">Name of the database of the container to monitor for changes</param>
+        /// <param name="containerName">Name of the container to monitor for changes</param>
+        public CosmosDBTriggerAttribute(string databaseName, string containerName)
         {
-            if (string.IsNullOrWhiteSpace(collectionName))
+            if (string.IsNullOrWhiteSpace(containerName))
             {
-                throw new ArgumentException("Missing information for the collection to monitor", "collectionName");
+                throw new ArgumentException("Missing information for the container to monitor", "container Name");
```

Comment:
The parameter name does not have a space in it. It is a single word.

You could use the `nameof` expression.
`nameof(containerName)`

Reply:
Interestingly, the other two classes don't make this check or throw any exceptions. I'll make a note to look into why that is.